### DISCUSSION TITLE
Centralized or Decentralized?

### DISCUSSION THREAD

**Discussion by kenricnelson** [cardano-foundation/CIPs#503] [General]
Achieving and maintaining decentralization is a challenging problem.  Decentralization requires soliciting broad input from a community and processing that input into high-quality decisions. The decisions need to a) strengthen the decentralized community to maintain competitiveness and b) prohibit capture by either internal or external actors. This discussion thread is intended to critically examine CIP-1694 in terms of its initiation of decentralized decision-making and the projected outcomes of the proposed processes.

**Comment by kenricnelson** [comment]
As a starting point, I wrote an essay examining the potential of CIP-1694 to empower a secret plutocracy.  https://medium.com/@photrek/cardano-improvement-proposal-1694-pathway-to-a-secret-plutocracy-f195b1666581

This first essay examines the ability to implement quadratic voting without identity verification. Quadratic voting is known to strengthen decentralized decision-making relative to one coin one vote. It is presumed to require identity verification; however, given that CIP-1694 specifies that the MVP will not have ID, a question arises regarding the relative merits of 1c1v and QV without ID.

A second essay is in draft regarding the importance of consent in group decision-making.  The second essay will have a team of co-authors.  If you are interested in co-authoring or commenting on the second essay, contact Kenric Nelson.

**Comment by kenricnelson** [comment]
As explained in the above essay 1c1v can't be regarded as a decentralized approach to governance given the ability of a few individuals to control the decision-making.  An even bigger vulnerability for CIP-1694 is the collusion designed into the three branches of government.  Given the unwillingness to implement any ID control, multipool SPOs are well-positioned to be large DReps. Given a controlling position as a large SPO and large DRep these individuals will appoint themselves to the Constitutional Committee.  

For separate branches of government to provide checks and balances they have to be independent.  Contradicting this basic requirement CIP-1694 calls for the three branches of government to be staffed by the same people. The inevitable collusion doesn't even require collaboration. It's just self-dealing for self-interest through and through.

This obvious vulnerability highlights the mistaken assumption that decentralized governance can exist without identification. Permissionless is fine for users of the network because the protocol is designed to restrict the manipulation of the tokens. Permissionless for the SPOs is okay in the short term but in the long term, wealth is not a sufficiently restrictive resource to guarantee that a) the 50% vulnerability threshold can't be met and that b) probabilistic attacks can't be executed at thresholds below 50%.  For this reason, the saturation of profits for SPOs was a critical design feature for Cardano, and a mechanism to saturate the profits of multipool SPOs is a critical requirement to fix the networks current vulnerability.

Governance cannot be restricted by a network protocol (users) or by running a particular software package (SPOs). If the same small group of individuals have controlling positions via 1c1v in both the SPO and DReps branches and also serve on the Constitutional Committee then it is quite feasible to write open-ended changes to either the network or the treasury and get them approved.

**Comment by ltouro** [comment]
I see a lot of critiscism to 1c1v but no viable alternative proposed. Would love to hear a feasible proposition to change that. One that is not full of its own (and even bigger) problems. Meanwhile, Cardano is indeed a plutocracy, for good and bad.

Regarding your proposal, I don't think a Captcha system would be a more sound mechanism to protect our governance. I don't understand how your suggestion to a 'pratical limit on the number of wallets' would work at the protocol level. Also, the correlation coefficient seems not only complex but any mechanism [public] designed to do so would be quickly gamed.

My conclusion is that being financially vested in the protocol is the (current) best way to have a proper governance. I would not invest a billion coins into something I'm trying to sabotage.
